testing/unit_tests/test_HeMIS.py

```diff
@@ -40,9 +42,9 @@ def setup_function():
     "extensions": [".nii.gz"],
     "roi_params": {"suffix": "_seg-manual", "slice_filter_roi": None},
     "contrast_params": {"contrast_lst": ['T1w', 'T2w', 'T2star']
-    }}])
+                        }}])
```

Comment:
was this indentation added by mistake?

Reply:
Yeah, auto format added that.  I updated it and moved it back... odd ending from previous line got carried over.